data/json/items/book/abstract.json

```
@@ -167,6 +167,24 @@
     "copy-from": "book_nonf_soft_tpl",
     "relative": { "price": 400, "price_postapoc": -2600, "intelligence": 3 }
   },
+  {
+    "abstract": "book_pulp_newage_tpl",
+    "type": "BOOK",
+    "name": { "str": "Generic New-Age Book" },
+    "description": "This is a template for pulp new-age books. They're theoretically nonfiction.",
```

As a convention, you need to put two spaces between sentences in player facing texts such as item descriptions:

```suggestion
    "description": "This is a template for pulp new-age books.  They're theoretically nonfiction.",
```

---

This applies to other new item descriptions you added as well.